pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/BundleSplitterTask.java

```diff
@@ -94,7 +94,7 @@ public BundleSplitterTask() {
                         if (bundleCount < maxBundleCount) {
                             bundleCache.add(bundle);
                         } else {
-                            log.warn(
+                            log.info(
```

Comment:
Can we make this a `debug` log? This feels like debug information, considering there isn't really an action for the end user, unless they want to increase their bundle limit. Operators can easily determine the number of bundles for the namespace at any time.

cc @merlimat @codelipenghui

Reply:
I created https://github.com/apache/pulsar/pull/14683 to continue this discussion.